src/chainstate/burn/db/sortdb.rs

```diff
@@ -597,14 +597,15 @@ const SORTITION_DB_INITIAL_SCHEMA: &'static [&'static str] = &[
     );"#,
     "CREATE INDEX canonical_stacks_blocks ON canonical_accepted_stacks_blocks(tip_consensus_hash,stacks_block_hash);",
     "CREATE TABLE db_config(version TEXT NOT NULL);",
-    r#"
+];
```

Comment:
There should only ever be one row in `db_config`, and it should be the current version.  Can you make the schema for `db_config` to be `db_config(version TEXT PRIMARY KEY)` instead, and use `INSERT OR REPLACE` when updating the version?